src/lib/config.ts

```diff
@@ -773,6 +773,11 @@ class default_config {
      *
      */
     perfsamples: string = "10000"
+
+    /**
+     * Whether or not to integrate with the TST tab
```

Comment:
You should explain what sort of integration there is (i.e, tab changing) and also that nothing bad will happen if TST is not installed.